### DISCUSSION TITLE
If a field is nullable and the type does not match then set value to null

### DISCUSSION THREAD

**Discussion by vinayak-idc** [rrousselGit/freezed#936] [Ideas]
Let suppose you are calling an api and parsing using freezed
now if there are some changes in backend related to datatype
   like :- id - before it was string now its int
our code will crash here, so can we check that is the data type do not match and our data type is nullable then it will print warning if in debugging mode and in app it will show null value or default value if there.

**Comment by rrousselGit** [comment]
I assume you're talking about fromJson?

That's not Freezed's responsibility. You can do that on your own by specifying a JsonConverter.
